src/main/resources/js/sourcegraph-bitbucket.js

```diff
@@ -0,0 +1,24 @@
+AJS.toInit(() => {
+    // Fetch Sourcegraph URL on page load
+    AJS.$.ajax({
+        url: AJS.contextPath() + '/rest/sourcegraph-admin/1.0/',
+        dataType: 'json'
+    }).done(({ url }) => {
+        if (!url) {
+            console.log('No Sourcegraph URL set')
```

Comment:
If a user sees this message, it might be useful to include a link to instructions on how to set the Sourcegraph URL.